install/install.go

```diff
@@ -0,0 +1,191 @@
+package install
+
+import (
+	"archive/zip"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+)
+
+func valid(module string) bool {
+	return strings.HasPrefix(module, "github.com/")
+}
+
+func Install(module string) {
+
+	if !valid(module) {
+		fmt.Printf(`Error reading URL. Please use "github.com/USER/REPO" format to install`)
+		return
+	}
+
+	err := getZip(module)
```

Comment:
It would be nice to print out the stages we're at:

* downloading ([with %](https://gist.github.com/albulescu/e61979cc852e4ee8f49c))
* unpacking
* creating alias
* done...

Reply:
The percent part needs some discussion 😄 it seems like HEAD github zip header does not return content-length so we need to think about a work-around

```
➜  ~ curl https://github.com/mingwho/abs/archive/master.zip
<html><body>You are being <a href="https://codeload.github.com/mingwho/abs/zip/master">redirected</a>.</body></html>%

➜  ~ curl -I https://codeload.github.com/mingwho/abs/zip/master
HTTP/1.1 200 OK
...
Content-Type: application/zip
Content-Disposition: attachment; filename=abs-master.zip
...
```

Reply:
Let's forget about this for now then -- let's just use a CLI loader like this:

```go
package main

import (
	"fmt"
	"time"
)

func main() {
	symbols := []string{"🌑 ", "🌒 ", "🌓 ", "🌔 ", "🌕 ", "🌖 ", "🌗 ", "🌘 "}
	i := 0

	for true {
		fmt.Printf("\r" + symbols[i] + " - downloading archive")
		time.Sleep(100 * time.Millisecond)
		i++
		if i > len(symbols)-1 {
			i = 0
		}
	}
}
```

Later on we can figure out how to print the progress bar :)

Reply:
WOW THIS IS COOL 👍

Reply:
Added CLI loader for download archive!